pynear/include/VPTree.hpp

```diff
@@ -69,18 +76,20 @@ template <typename T, typename distance_type, distance_type (*distance)(const T
 
     VPTree(const std::vector<T> &array) { set(array); }
 
-    void set(const std::vector<T> &array) {
+    void set(const std::vector<T> &array) { set(std::move(array)); }
```

Comment:
this doesn't work right? move from const? also it would be confusing for the user if a fucntion with this signature would move.

Reply:
Oh, yes, will change. Thanks.

Reply:
Changed on 30c3bc5650e2686902a6ef053336371f1bd86215 and [c8ea04f](https://github.com/pablocael/pynear/pull/48/commits/c8ea04f1b4a2eabb88d87514dec36087e32ff74a)